terraform/aptos-node/aws/cluster.tf

```diff
@@ -43,14 +43,14 @@ locals {
       min_size      = var.utility_instance_min_num
       desired_size  = var.utility_instance_num
       max_size      = var.utility_instance_max_num > 0 ? var.utility_instance_max_num : 2 * var.utility_instance_num
-      taint         = false
+      taint         = var.utility_instance_enable_taint
```

Comment:
I think it'd call this `validator_nodepool_enable_taint`

Reply:
wanted to keep the same naming scheme `<nodegroup>_<instance>_<var>`, despite being slightly inaccurate (instance vs nodepool).